integ-test/src/test/java/org/opensearch/sql/calcite/remote/fallback/CalciteLikeQueryIT.java

```diff
@@ -16,4 +19,53 @@ public void init() throws Exception {
     super.init();
     enableCalcite();
   }
+
+  @Override
+  @Test
+  @Ignore("* in like is handled wrong")
+  public void test_like_with_escaped_percent() throws IOException, IOException {
+    super.test_like_with_escaped_percent();
+  }
```

Comment:
the entire class was ignored, why add following code? do you want to remove the line L15

Reply:
Yes. I remove the line 15 so we can pass 1 test.